pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -366,19 +440,36 @@ type reconstructedVolume struct {
 	blockVolumeMapper   volumepkg.BlockVolumeMapper
 }
 
+type globalVolumeInfo struct {
```

Comment:
why it is called globalVolumeInfo? maybe add comment?

Reply:
added